Kernel/Tasks/Scheduler.cpp

```diff
@@ -375,7 +375,7 @@ UNMAP_AFTER_INIT void Scheduler::initialize()
     auto [colonel_process, idle_thread] = MUST(Process::create_kernel_process(KString::must_create("colonel"sv), idle_loop, nullptr, 1, Process::RegisterProcess::No));
     s_colonel_process = &colonel_process.leak_ref();
     idle_thread->set_priority(THREAD_PRIORITY_MIN);
-    idle_thread->set_name(KString::must_create("Idle Task #0"sv));
+    MUST(idle_thread->set_name(KString::must_create("Idle Task #0"sv)));
```

Comment:
That can now use a static TaskName

Reply:
It will use a TaskName, but it will not be static. I will try to see how to fix this now.